amr-wind/wind_energy/ABLStats.H

```diff
@@ -4,6 +4,7 @@
 #include <memory>
 
 #include "amr-wind/CFDSim.H"
+#include "amr-wind/utilities/ncutils/nc_interface.H"
```

Comment:
If you eliminate `calc_sfs_stress_avgs` as a method you can eliminate this include

Reply:
Addressed in https://github.com/Exawind/amr-wind/pull/290/commits/da05df00d9686f43b32742e3ec13693811db1ab3